source/dub/commandline.d

```diff
@@ -21,7 +21,7 @@ import dub.packagesuppliers;
 import dub.project;
 import dub.internal.utils : getDUBVersion, getClosestMatch, getTempFile;
 
-import dub.internal.dyaml.stdsumtype;
```

Comment:
I think moving this out of the dyaml package is unnecessary. I'd rather keep the diff with upstream `dyaml` low than have a more "correct" placement. There's no tangible benefit in the move, but some tangible downsides (when I sync things, I usually just cp files over and check the diff to see if some adaptations still make sense).